Xwt.Gtk/Xwt.GtkBackend/LabelBackend.cs

```diff
@@ -168,6 +166,7 @@ public void SetFormattedText (FormattedText text)
 			if (links != null)
 				links.Clear ();
 
+			var indexer = new TextIndexer (text.Text);
```

Comment:
Would it make sense to re-use the TextIndexer? The [ApplyFormattedText](https://github.com/mono/xwt/blob/44369a21f3fe3127b325e8c0d61d83da15f1e8e4/Xwt.Gtk/Xwt.GtkBackend/Util.cs#L296) method creates its own TextIndexer. Could we also allow it to be passed to ApplyFormattedText?

Reply:
yeah, fixed now to do that